src/languages/en.ts

```diff
@@ -1571,51 +1571,51 @@ export default {
+        hasPhoneLoginError: 'To add a bank account, please ensure your primary login is a valid email and try again. You can add your phone number as a secondary login.',
         hasBeenThrottledError: 'There was an error adding your bank account. Please wait a few minutes and try again.',
-        hasCurrencyError: 'Oops! It appears that your workspace currency is set to a different currency than USD. To proceed, please set it to USD and try again.',
+        hasCurrencyError: "Oops! It looks like your workspace currency isn't set to USD. Please set it to USD and try again.",
         error: {
-            youNeedToSelectAnOption: 'You need to select an option to proceed.',
-            noBankAccountAvailable: 'Sorry, no bank account is available.',
-            noBankAccountSelected: 'Please choose an account.',
-            taxID: 'Please enter a valid tax ID number.',
-            website: 'Please enter a valid website. The website should be in lowercase.',
-            zipCode: `Incorrect zip code format. Acceptable format: ${CONST.COUNTRY_ZIP_REGEX_DATA.US.samples}`,
-            phoneNumber: 'Please enter a valid phone number.',
-            companyName: 'Please enter a valid legal business name.',
-            addressCity: 'Please enter a valid city.',
-            addressStreet: 'Please enter a valid street address that is not a PO box.',
-            addressState: 'Please select a valid state.',
-            incorporationDateFuture: 'Incorporation date cannot be in the future.',
-            incorporationState: 'Please select a valid state.',
-            industryCode: 'Please enter a valid industry classification code. Must be 6 digits.',
-            restrictedBusiness: 'Please confirm company is not on the list of restricted businesses.',
-            routingNumber: 'Please enter a valid routing number.',
-            accountNumber: 'Please enter a valid account number.',
-            routingAndAccountNumberCannotBeSame: 'The routing number and account number cannot be the same.',
-            companyType: 'Please select a valid company type.',
-            tooManyAttempts: 'Due to a high number of login attempts, this option has been temporarily disabled for 24 hours. Please try again later or manually enter details instead.',
-            address: 'Please enter a valid address.',
-            dob: 'Please select a valid date of birth.',
-            age: 'Must be over 18 years old.',
-            ssnLast4: 'Please enter valid last 4 digits of SSN.',
-            firstName: 'Please enter a valid first name.',
-            lastName: 'Please enter a valid last name.',
-            noDefaultDepositAccountOrDebitCardAvailable: 'Please add a default deposit bank account or debit card.',
-            validationAmounts: 'The validation amounts you entered are incorrect. Please double-check your bank statement and try again.',
+            youNeedToSelectAnOption: 'Please select an option to proceed',
```

Comment:
@jamesdeanexpensify Was it intentional to remove the period after the error messages? 

I can see a rule in our codebase `Use periods at the end of error messages.`

Reply:
Sorry, I can see the PR is reverted. You can ignore my previous comment.

Reply:
Yeah, that's actually not accurate. This is where we landed internally, so I'd love to change that rule if someone knows how: 
<img width="624" alt="image" src="https://github.com/Expensify/App/assets/22160301/3f699d42-882c-42aa-bbbb-1cccfb610e83">

Reply:
I am not sure how this rule is configured and whether it has an option to set based on line/sentences. Quick fix is:

You could add `// eslint-disable-next-line rulesdir/use-periods-for-error-messages` before every line.

Reply:
i might hold off on doing anything for now - i'm not an engineer and i don't want to mess with anything! haha